idlgen/src/lib.rs

```diff
@@ -340,6 +248,9 @@ service {
   query That : (pr1: SailsIdlgenTestsThatParam) -> str;
 }
 ";
+
         assert_eq!(generated_idl, EXPECTED_IDL);
+        // assert!(generated_idl_program.is_ok());
+        // assert_eq!(generated_idl_program.unwrap().items.len(), 8);
```

Comment:
Dead code?

Reply:
Not quite. It waits for [this PR](https://github.com/gear-tech/sails/pull/35). Going to merge it and uncomment this